week2/exercise P1008 in LuoGu.cpp

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+int main()
+{
+	int num1, num2, num3;
+	for(int i = 1; i < 10; i++) {
+		for(int j = 1; j < 10; j++) {
+			if(i == j) {
+				continue;
+			} else {
+				for(int k = 1; k < 10; k++) {
+					if(i == k || j == k) {
+						continue;
+					} else {
+						num1 = i * 100 + j * 10 + k; // Get a three-digit number that is different for each digit.
+						num2 = num1 * 2;
+						num3 = num1 * 3;             // Get three numbers in the ratio of 1:2:3
+						if(num2 % 10 == num2 / 10 % 10 
+						|| num2 / 100 == num2 / 10 % 10 
+						|| num2 % 10 ==  num2 / 100
+						|| num3 % 10 == num3 / 10 % 10 
+						|| num3 / 100 == num3 / 10 % 10 
+						|| num3 % 10 ==  num3 / 100
+						|| num3 >= 1000) {

```

Comment:
We can define a function `isValid` that takes a single integer as a parameter to determine what number is a valid number:
1. The number should be less than 1000
2. The number should be no less than 100
3. Each digit of the number should be different.

Then we can use the function to determine whether `double` and `triple` is a valid number or not (Even the `num` actually). In this way we can improve code readability and reusability.